### ISSUE TITLE
[Chromaticraft] New Mod Conflict on v29 builds - HexCraft

### ISSUE THREAD

**Issue by Milii20** [ReikaKalseki/Reika_Mods_Issues#2798]
TLDR: Researching which mod makes Chromaticraft v29e crash, will update as soon as i have something

Greetings!, aparently i am the bearer of bad news and/or crash reports. After the config issue was fixed, i went to update my current modpack and i am finding a odd crash report. I have to add it worked in v28 (and still works if i go back to that version) 

https://pastebin.com/g6iS3sQi

I have to add: I have an old world and old configs, but i am testing this on a new world and freshly generated configs just to be sure.
I think it has something to do with this changelog on v29a? not sure tho:
"ChromatiCraft: Fixed some "mod interface" casting recipes not showing in the book or NEI"

Since the problem is not Chromaticraft itself, but something related to other mods, i am still researching which mod actually causes Chromaticraft to crash and will update as soon as possible.

Thank you very much!

**Comment by Milii20** [comment]
Nevermind, actually has something to do with this changelog i think:

"ChromatiCraft: Fixed missing items in ItemMagicRegistry (elemental assignment)"

**Comment by ReikaKalseki** [comment]
You have hexcraft installed, and its crystal block ID is null. Why?

**Comment by Milii20** [comment]
You are totally right, the problem is totally Hexcraft, but it's the latest version from the curseforge page (0.13.2, i redownloaded just to be sure, and also tested other versions and the github 0.13.2) and removing it seems to not crash anymore.

But i have no idea why it has a null block ID
Would it make sense if it had something to do with Chromaticraft v29? 
or probably was also broken on v28 but there was no crash since it didnt launch an exception?

**Comment by Milii20** [comment]
Okay, at this point i am sorry to even bother you, but i found a weirder crash related to Chromaticraft. 
It happen while generating a new world once i took HexCraft out, and since it's kinda a mod conflict, i wont make a new report.

https://pastebin.com/RnRtm6f7

I think what happen here is one of two options, it either tried to generate something from Chromaticraft on top (or at the same coordinates) of a pam's harvestcraft tree, or viceversa.
This is way harder to repeat, and going back to my old world aparently fixes it since it's already generated.
Don't really worry about this one, but since it was a weird crash i am posting it here


EDIT: okay, confirmed, was preciselly my seed, changed the seed and suddenly no more crash

**Comment by ReikaKalseki** [comment]
This issue is entirely unrelated to the last one, and is a copy of #2797.

For your first issue, I need you to re-add hexcraft and then send me the entire loading log from an attempted launch.

**Comment by Milii20** [comment]
Here is the log of the entire modpack with HexCraft:

fml-server-latest: 
[fml-server-latest.log](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/6224932/fml-server-latest.log) 
(Pastebin wont accept the log since it's too big)

**Comment by ReikaKalseki** [comment]
Fixed.

**Comment by Milii20** [comment]
Thank you very much!
